Gemfile

```diff
@@ -51,3 +62,4 @@ gem 'minitest-reporters'
 gem 'minitest-autotest'
```

Comment:
If we're adding the test/development groups, can you add tabs before this and the `simplecov` lines?